docs/source/content/supervised/logistic_regression.rst

```diff
@@ -18,39 +18,52 @@ Here is the standard logistic function, note that the output is always between 0
 3      False    True 
 0.5    True     False 
 2      False    True 
-=====  ======   =======
+=====  =======  =======
 
-Notice that the student's success is determined by the inputs and the value is binary, so Logistic Regression will work well for this scenario.
+Notice that the student's success is determined by the inputs and the value is binary, so logistic regression will work well for this scenario.  
 
 
 How does it work?
 -----------------
 
-Logistic regression works using a linear combination of inputs, so multiple information sources can govern the output of the model.  The parameters of the model are the weights of the various features, and represent their relative importance to the result.
+Logistic regression works using a linear combination of inputs, so multiple information sources can govern the output of the model.  
+The parameters of the model are the weights of the various features, and represent their relative importance to the result.  
+In the equation that follows, you should recognize the formula used in linear regression.  
+Logistic regression is, at its base, a transformation from a linear predictor to a probability between 0 and 1.
 
-.. figure:: _img/Logistic_Eq.svg
+.. figure:: _img/WikiLogisticEQ.svg 
    :alt: Equation
 Ref: https://en.wikipedia.org/wiki/Logistic_regression
 
-The beta values are the weights and x values are the variable inputs.  This formula gives the probability that the input belongs to Class B, which is the output of the function.
+As in linear regression, the beta values are the weights and x values are the variable inputs.  
+This formula gives the probability that the input belongs to Class B, which is the goal of the logistic regression model.
+
 
 Conclusion
 -----------------
 
-Machine learning libraries make using Logistic Regression very simple.  Check out the example code in the repository and follow along. The basic idea is to supply the training data as pairs of input and classification, and the model will be built automatically.  As always, keep in mind the basics mentioned in this repository, as there is no fool-proof method for machine learning.
+Logistic regression build upon linear regression by extending its use to classification.  
+Although it is not able to classify into more than two classes, it is still effective in what it does, and simple to implement.  
+Consider logistic regression as the first thought pass/fail method.  
+When you just need a pass/fail probability from data, logistic regression is the simplest and likely best option.
+
+Machine learning libraries make using Logistic Regression very simple.  Check out the example code in the repository and follow along. 
+The basic idea is to supply the training data as pairs of input and classification, and the model will be built automatically.  
+As always, keep in mind the basics mentioned in the overview section of this repository, as there is no fool-proof method for machine learning.
```

Comment:
Add motivation section and add code snippets and show how it relates to the doc.